sec-priv-q.md

```diff
@@ -0,0 +1,298 @@
+cross-site and the epoch design relies on values that are stable for an entire
+epoch (a week).  This makes state-clearing events, particularly those that are
+narrowly targeted, somewhat challenging to reason about.  We haven't worked
+through all of the requirements and constraints in this area, such that we might
+infer the correct design.
+
+### Do the features in your specification expose information about the
+    underlying platform to origins?
+
+No.  However, if a compatible system is developed in the underlying platform for
+the management of match keys, user agent implementations might move to unify the
+two sources of match keys.  This would enable attribution that crosses from the
+web into applications.  No such mechanism currently exists.
+
+### Does this specification allow an origin to send data to the underlying platform?
+
+No.
+
+### Do features in this specification enable access to device sensors?
+
+No.
+
+### Do features in this specification enable new script execution/loading
+    mechanisms?
+
+No.
+
+### Do features in this specification allow an origin to access other devices?
+
+No.
+
+### Do features in this specification allow an origin some measure of control
+    over a user agent’s native UI?
+
+No.
+
+### What temporary identifiers do the features in this specification create or
+    expose to the web?
+
+The match key itself is not exposed to the web, so this question does not apply.
+
+The ciphertext of the encrypted match key provided to sites is stable over a set
+period.  This might be used as an identifier.  However, the lifetime of this
+value is bound to the storage lifetime for the site that receives it.  A fresh,
+unlinkable encryption is generated if site storage is cleared (and the
+underlying value might also need to be randomized too, though this question
+still hasn't been completely resolved; see above).
+
+### How does this specification distinguish between behavior in first-party and
+    third-party contexts?
+
+Information provided by the API is only provided to the top-level browsing
+context (a.k.a., the "first-party context").  Other sites that contribute to the
+content of a web page only receive a copy of the value provided to the top-level
+site, but only if the top-level site permits it.
+
+### How do the features in this specification work in the context of a browser’s
+    Private Browsing or Incognito mode?
+
+This question is not entirely resolved.  User agents that want maximum privacy
+protection - which seems likely in this particular case - can randomize the
+match key that is provided to sites.  This ensures that attribution is not
+possible based on the information that is provided.  This behaviour is
+indistinguishable to a site from use of a real match key and so does not offer
+sites a reason to discriminate against users.
+
+The same strategy can be employed for users who opt out of use of the API.
```

Comment:
Maybe worth adding a bit more on when to generate random matchkeys and when to cache their encryptions. 

An opt-out user agent will generate a random matchkey (preferably in some large set of opt-out/dummy matchkeys) for each site that requests it and cache the encryption of it for an epoch.  Different sites will have different matchkeys so there won’t be a chance of attribution.  Caching for each site over an epoch is necessary to ensure that the opt-out choice is not visible to sites.